setup.cfg

```diff
@@ -10,10 +10,9 @@ long_description_content_type = text/x-rst
 classifiers =
     Development Status :: 5 - Production/Stable
     License :: OSI Approved :: Apache Software License
-    Programming Language :: Python :: 2.7
-    Programming Language :: Python :: 3.6
```

Comment:
I do have one reservation: here we are silently dropping support for 2.7 and 3.6.  This should at least be a separate commit, and possibly even a PR to discuss.

I presume the current version of this code still works with 2.7.  Let's at least _say_ we're dropping support rather than silently ditching it!

Reply:
There is a `Remove python 2.7 support` commit. Do you mean you don't like that it also drops 3.6?

The code may well work on 2.7, the problem is just getting is flake8 dependency problems across python versions.

Reply:
Oh, sorry about that, I was looking at the overall changes and didn't check the commit list.

Just change the commit message to mention 3.6 at the same time and all is good.